rpcs3/rpcs3qt/gs_frame.cpp

```diff
@@ -299,6 +304,70 @@ void gs_frame::flip(draw_context_t, bool /*skip_frame*/)
 	}
 }
 
+void gs_frame::take_screenshot(std::vector<u8> ss_data, const u32 ss_width, const u32 ss_height)
+{
+	std::thread(
+	    [ss_width, ss_height](std::vector<u8> sshot_data) {
+		    std::string screen_path = fmt::format("%s/Screenshots/", fs::get_config_dir());
```

Comment:
Can use const and concat here. Also using lowercase screenshot is probably better